src/renderer/entities/transaction/lib/common/utils.ts

```diff
@@ -106,16 +106,14 @@ export const isEditFlexibleTransaction = (transaction?: Transaction | DecodedTra
       return false;
     }
 
-    const removeProxyTransaction = transaction.args.transactions.at(0);
```

Comment:
I suggest to ado early return when checking for BATCH_ALL so there will be one less nested layer:

```ts
if (transaction?.type !== TransactionType.BATCH_ALL) {
return false;
}

// The rest
``